reproman/support/jobs/orchestrators.py

```diff
@@ -664,13 +668,19 @@ def failed_subjobs(self):
                     # All right, something looks off.
                     raise exc
             else:
-                failed = list(map(int, failed_ref.strip().split())) or failed
+                # Line format: mode type object filename
+                failed = [int(ln.split()[3])
```

Comment:
What happens with filename with spaces? Should split get limited?

Reply:
This subtree is controlled by us, and we write files with integer names to it.

> What happens with filename with spaces?

If a file like "3 2" ended up in there, we'd mistakenly consider job 3 a failed job.  The larger problem though would be how the file name "3 2" got there.

> Should split get limited?

I don't think it matters.  If someone is tampering with `<metadirectory>/failed/`, I'd say all bets are off.  But sure, we might as well.  It doesn't cost anything to ensure we get the full file name and in some cases has a better response to the unexpected invalid file (crash rather take the leading integer silently).

I'll instead update it to split on tab because the format is `mode<SP>type<SP>object<TAB>filename`.  In the scenario above, `int("3 2")` would then fail with a ValueError.  We don't need to worry about "3<TAB>2" because Git will convert the tab to "\t".